web/srv/api_handlers.go

```diff
@@ -434,11 +434,44 @@ func (h *handler) handleAPIResourceDefinition(w http.ResponseWriter, req *http.R
 	w.Write(resourceDefinition)
 }
 
-func (h *handler) handleGetExtension(w http.ResponseWriter, req *http.Request, _ httprouter.Params) {
+func (h *handler) handleGetExtensions(w http.ResponseWriter, req *http.Request, _ httprouter.Params) {
 	ctx := req.Context()
 	extensionName := req.FormValue("extension_name")
 
 	resp := map[string]interface{}{}
+	if extensionName == "" {
+		extensions, err := h.k8sAPI.GetAllNamespacesWithExtensionLabel(ctx)
+		if err != nil {
+			renderJSONError(w, err, http.StatusInternalServerError)
+			return
+		}
+
+		type Extension struct {
```

Comment:
I think we're missing the `namespace` here? looking at issue, it seems the namespace should be present for the extension. Is this something that we want to display?

cc. @dadjeibaah